try:
    let_us_cause_a_NameError
except NameError, err:
    print err, '--> our Python 2 error message'
